### PR TITLE
XLS-37d: Improved Concise Transaction Identifier (CTID)

### PR DESCRIPTION
**Pull request by dangell7**
- Discussion: https://github.com/XRPLF/XRPL-Standards/discussions/91
- Proposed implementation: https://github.com/XRPLF/rippled/pull/4418

### CODE REVIEW

File: XLS-37d/README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+```
+Title:       Improved Concise Transaction Identifier (CTID)
+Revision:    1 (2023-02-11)
```
Comment (Silkjaer) [inline XLS-37d/README.md:3]: Revision should be removed.

File: XLS-37d/README.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+Authors:     Richard Holland (XRPL-Labs, XRPLF)
+             Ryan Molley
+```
```
Comment (Silkjaer) [inline XLS-37d/README.md:6]: Type: Draft should be added.

File: XLS-37d/README.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
```
Comment (Silkjaer) [inline XLS-37d/README.md:1]: Folder name should be updated to reflect `XLS-0000d-{title}` where title is lowercase and spaces replaced by a hyphen.

Comment (dangell7) [reply XLS-37d/README.md]: Do you mind if I shorten the titles? Or is `concise-transaction-identifier` what you're looking for?

**Comment by intelliot** [comment]
In the spec, I recommend including a link to the proposed implementation: https://github.com/XRPLF/rippled/pull/4418

**Comment by intelliot** [comment]
@dangell7 @RichardAH Can you add a paragraph explaining why and how this feature improves the user experience for sidechains?

**Comment by dangell7** [comment]
@intelliot 

```
Improved Concise Transaction Identifiers (CTIDs) improve the user experience for sidechains because they provide a way to efficiently locate a specific transaction on a specific chain without relying on transaction hashing, which can be resource-intensive and time-consuming. CTIDs allow users to quickly and easily identify their transactions on a particular chain using its ledger sequence number, transaction index, and network ID, making it easier for them to track their transactions and confirm their successful completion. Additionally, CTIDs require less storage space than transaction hashes, which can be beneficial for databases containing millions of transactions.
```

File: XLS-37d-ctid/README.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+  }
+"hash":"53726E6D021522A97602004841A2A7477D1900BD348ADA4D4DF812B466E56454"
+"ledger_index":62084722
+"date":1615278490000
+}
+```
+
+### 1.3 Motivation
+
+The XRP Ledger is poised to become (or depending on the time of reading: has already become) an ecosystem of cooperatively interconnected XRPL Protocol Chains. It is imperative that users of these chains can efficiently locate a specific transaction on a specific chain. Therefore a network-aware transaction identifier is necessary.
```
Comment (intelliot) [inline XLS-37d-ctid/README.md:87]:
```suggestion
The XRP Ledger can support an ecosystem of cooperatively interconnected XRPL Protocol Chains. Users of these chains need to efficiently locate specific transactions on specific chains.

The CTID is a network-aware transaction identifier which improves the user experience for sidechains. It provides a way to efficiently locate a specific transaction without relying on transaction hashes, which can be difficult to find in a multi-chain environment.

CTIDs allow users to quickly and easily identify their transactions on a particular chain using the ledger sequence number, transaction index, and network ID. Users can thus identify their transactions and confirm their successful completion. Additionally, CTIDs require less storage space than transaction hashes, which can be beneficial for databases containing millions of transactions.
```

**Review by intelliot — APPROVED** [review]

**Pull request merged**